GVFS/GVFS.PlatformLoader/PlatformLoader.Mac.cs

```diff
@@ -1,17 +1,10 @@
 using GVFS.Common;
-using GVFS.Common.Git;
 using GVFS.Platform.Mac;
-using GVFS.Virtualization.FileSystem;
 
 namespace GVFS.PlatformLoader
 {
-    public static class GVFSPlatformLoader
+    public static partial class GVFSPlatformLoader

```

Comment:
Are the `PlatformLoader` changes still needed (now that FastFetch has the native calls for both platforms in its own project)?

Reply:
It's not needed as part of this PR after pulling code directly into FastFetch. However, I think this change is still virtuous, although it's rather small. Would you rather me split this off into a separate PR?

Reply:
> I think this change is still virtuous

What's the benefit of this change?

> Would you rather me split this off into a separate PR?

I'd rather this change be its own small PR.  This PR is large enough that it will be helpful to restrict the changes to only those needed to enable FastFetch.